package.json

```diff
@@ -32,7 +32,10 @@
     "service-fabrik"
   ],
   "dependencies": {
```

Comment:
FOSS approvals required right ?

Reply:
Yes, will raise a FOSS approval request for the new dependencies